site/ecosystem.md

```diff
@@ -101,6 +101,7 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 - [Saite](https://github.com/jsa-aerial/saite) Interactive exploratory graphics and ad hoc visualization application for Clojure(Script). Built on top of [Hanami](https://github.com/jsa-aerial/hanami).
 - [Datablocks](https://datablocks.pro), a node-based editor for exploring, analyzing and transforming data without code.
 - [Rath](https://github.com/Kanaries/Rath) An augmented analysis tool including auto-EDA, pattern discovery, multi-dimensional visualization recommendation, and interactive dashboards generation.
+- [MarkText](https://github.com/marktext/marktext): A simple and elegant open-source markdown editor that focused on speed and usability.
```

Comment:
```suggestion
- [MarkText](https://github.com/marktext/marktext): A markdown editor that focuses on speed and usability.
```

Reply:
Oh, that just referenced from MarkText description. 
I just an user not the developer. 
I think we can simply use markdown editor as the description if we need more objective.